packages/zowe-explorer/CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 All notable changes to the "vscode-extension-for-zowe" extension will be documented in this file.
 
+## `1.15.0`
+
+- Added the secure credentials support for Extenders API [#1306](https://github.com/zowe/vscode-extension-for-zowe/pull/1306). Thanks @JillieBeanSim
+- Improved the Command Palette by adding 'Zowe Explorer:` before all commands that are related to the extension. Removed some commands from the palette that caused issues [#1308](https://github.com/zowe/vscode-extension-for-zowe/pull/1308). Thanks @lauren-li
+- Updated Theia Tests. Now you need to have Zowe CLI 6.30.0 and the latest .vsix file in the `theia/plugins` folder to run Theia tests [#1268](https://github.com/zowe/vscode-extension-for-zowe/pull/1268). Thanks @deepali-hub
+- Fixed the issue that preventied the `issue STOP command` function from executing correctly [#1304](https://github.com/zowe/vscode-extension-for-zowe/pull/1304). Thanks
+  @nickImbirev
+- Fixed the issue that caused the Add Profile icon to disapper [#1307](https://github.com/zowe/vscode-extension-for-zowe/pull/1307). Thanks @lauren-li
+- Fixed the vulnerability in NPM Audit [#1309](https://github.com/zowe/vscode-extension-for-zowe/pull/1309). Thanks @JillieBeanSim
+- Fixed the issue that doubled the occurence of the port prompt [#1298](https://github.com/zowe/vscode-extension-for-zowe/pull/1298). Thanks @katelynienaber
+- Fixed the issue that affected VS Code views after deleting a key binding in Zowe Explorer [#1310](https://github.com/zowe/vscode-extension-for-zowe/pull/1310). @crawr
```

Comment:
This is a bit confusing for me, but I am not sure if this is more confusing: "Fixed the issue that affected other VS Code views when using the delete key because of the key binding in Zowe Explorer"

Reply:
Ok, let's try to break this down. What was the issue? and what was causing this issue? Let me try to answer these.
The issues: "other VS Code views" are affected.
The cause of the issues: A user deleted a key binding in Zowe Explorer
This is not super clear to me and I want to ask the following questions: 
1.  What is "VS Code view"?
2. What are other VS Code views? 
3. How does one delete a key binding in Zowe Explorer?

Based on the answers, I might improve wording :)

Reply:
The issue is that if I use the `Delete` key anywhere in VS Code, even outside ZE, it triggers the delete command in Zowe Explorer. The fix is to limit the delete command to only the ZE jobs view

Reply:
Ok, so, 'view' refers to those dialogs (sections) for DS/USS/Jobs. Let me change the original sentence to the following one: 
"Fixed the issue that affected VS Code views (dialogs with Data sets, USS, and jobs) after using the `delete` command in Zowe Explorer" Does that reflect reality a bit better?

Reply:
It did not just affect those three views. In the [issue](https://github.com/zowe/vscode-extension-for-zowe/issues/1260) opened for this, you can see he is not even in Zowe Explorer but the Zowe job command is triggered when he pressed the Delete key in his keyboard.

Reply:
"Fixed the issue that affected other VS Code views, including dialogs with Data sets, USS, and jobs, after using the delete command in Zowe Explorer" - do you think this will work?

Reply:
How about "Fixed the issue that triggered the Delete Job command even outside Zowe Explorer views" ?

Reply:
I think this makes sense!